playwright.config.ts

```diff
@@ -19,20 +20,22 @@ const config = {
       name: 'chromium-wide',
       use: {
         ...devices['Desktop Chrome'],
-        viewportSize: { width: 1440, height: 900 },
+        headless: true,
+        viewport: { width: 1440, height: 900 },
       },
     },
     {
       name: 'chromium-mobile',
       use: {
         ...devices['Pixel 5'],
-        viewportSize: { width: 375, height: 667 },
+        headless: true,
+        viewport: { width: 375, height: 667 },
       },
     },
   ],
   webServer: {
     command: 'make start-integration CI=1',
-    url: 'http://localhost:6130/health',
+    url: 'http://localhost:6130',
```

Comment:
Why this change? `baseURL` at the top should supersede for tests, but this is used to determine if the webserver is ready

Reply:
I can revert this. None of the webserver stuff works locally I had to comment it all out.